src/helper/remote_config/asm_data_listener.cpp

```diff
@@ -155,7 +163,11 @@ void dds::remote_config::asm_data_listener::on_update(const config &config)
         }
     }
 
-    auto parameter = rules_to_parameter(rules_data);
-    auto parameter_view = dds::parameter_view(parameter);
-    engine_->update_rule_data(parameter_view);
+    std::map<std::string_view, std::string> meta;
+    std::map<std::string_view, double> metrics;
+
+    engine_ruleset ruleset = rules_to_engine_ruleset(rules_data);
+
+    // TODO find a way to provide this information to the service
```

Comment:
What do you mean with this?

Reply:
Updating the WAF results in new metrics and tags so eventually we need a mechanism to pass them to the next available client.